app/services/checksum_validator.rb

```diff
@@ -2,86 +2,81 @@
 
 # code for validating Moab checksums
 class ChecksumValidator
-  attr_reader :bare_druid, :results, :complete_moab
+  attr_reader :druid, :results, :complete_moab
```

Comment:
bare_druid vs. druid:  in some of our apps "druid" is the full druid.  bare_druid makes it clear that in this app we do not want the prefix.  I am fine with the will of the people on this decision.

Reply:
I agree this is a big mess. However, the attribute on PreservedObject is `druid` so this keeps it consistent. In fact, I'm going to change this to a `delegate`.